README.md

```diff
@@ -74,17 +74,22 @@ For details on the difference between the builds, see <a href="#which-build-is-r
 To load the "standard" build, import modules from the `web-vitals` package in your application code (as you would with any npm package and node-based build tool):
 
 ```js
-import {onLCP, onINP, onCLS} from 'web-vitals';
-
-onCLS(console.log);
-onINP(console.log);
-onLCP(console.log);
+import('web-vitals')
+  .then(({onCLS, onINP, onLCP}) => {
+    onCLS(console.log);
+    onINP(console.log);
+    onLCP(console.log);
+  })
+  .catch((error) => {
+    console.error('Failed to load web-vitals library:', error);
+  });
 ```
 
-> [!NOTE]
-> In version 2, these functions were named `getXXX()` rather than `onXXX()`. They've [been renamed](https://github.com/GoogleChrome/web-vitals/pull/222) in version 3 to reduce confusion (see [#217](https://github.com/GoogleChrome/web-vitals/pull/217) for details) and will continue to be available using the `getXXX()` until at least version 4. Users are encouraged to switch to the new names, though, for future compatibility.
+<a name="dynamic-imports"></a>
```

Comment:
As mention in my other comment, I don't think we need this note, unless we want to move it to its own dedicated section on code splitting.

Reply:
Ditto.